app/views/find-free-text.html

```diff
@@ -150,13 +150,13 @@ <h2 class="govuk-heading-l">Browse interventions by category</h2>
 
       <ul class="govuk-list">
         <li>
-          <a href="/results?category=high-risk-for-public">High risk for public</a>
+          <a href="/results?category=high-risk-for-public">High risk to public</a>
         </li>
         <li>
-          <a href="/results?category=high-risk-for-children">High risk for children</a>
+          <a href="/results?category=high-risk-for-children">High risk to children</a>
         </li>
         <li>
-          <a href="/results?category=aged-18-to-24">Aged 18 to 24</a>
+          <a href="/results?category=aged-18-to-25">Aged 18 to 25</a>
```

Comment:
@PabloCabello when you change the link text, please can you change the `href` to match? e.g. here changing to `aged-18-to-25`. I've done it for you on this PR.